### PR TITLE
Fix whitespace issues

### PR DESCRIPTION
**Pull request by marcoscaceres**
This change is [Reviewable](https://reviewable.io/reviews/w3c/respec/785)

### CODE REVIEW

File: js/core/utils.js [hunk 1]
```diff
@@ -43,21 +42,27 @@ define(
...
+                            return smallest;
+                        }
                         var match = item.match(spaceOrTab)[0] || "";
-                        return (match) ? match.length : 0;
-                    })
-                    .sort()
-                    .shift(1);
-                return leftPad || 0;
-            }
+                        return (match.length < smallest) ? match.length : smallest;
```
Comment (dontcallmedom) [inline js/core/utils.js]: Math.min ?

**Comment by dontcallmedom** [comment]
LGTM with a possible minor improvement; not sure why the Travis CI PR build failed (but the branch push seems to be happy)

File: js/core/utils.js (hunk 1 above)
Comment (marcoscaceres) [inline js/core/utils.js]: Ah yeah, better.

**Comment by marcoscaceres** [comment]
Travis fails intermittently even though everything passes 😢 . Running the tests again fixes it.

**Comment by marcoscaceres** [comment]
@dontcallmedom thanks for review!

**Pull request merged**